src/adapters/peggedAssets/usd-coin/index.ts

```diff
@@ -114,6 +114,10 @@ const chainContracts: ChainContracts = {
     bridgeOnETH: ["0x3980c9ed79d2c191A89E02Fa3529C60eD6e9c04b"],
     bridgedFromETH: ["0xea32a96608495e54156ae48931a7c20f0dcc1a21"],
   },
+  polygon_zkevm: {
+    bridgeOnETH: ["0x2a3dd3eb832af982ec71669e178424b10dca2ede"],
+    bridgeFromETH: ["0xa8ce8aee21bc2a48a5ef670afcc9274c7bbbc035"],
```

Comment:
What is this address? 
0xA8CE8aee21bC2A48a5EF670afCc9274C7bbbC035

Reply:
USDC on zkevm
https://zkevm.polygonscan.com/token/0xA8CE8aee21bC2A48a5EF670afCc9274C7bbbC035